shell/common/rasterizer.h

```diff
@@ -204,6 +204,14 @@ class Rasterizer final : public SnapshotDelegate,
 
   fml::TaskRunnerAffineWeakPtr<SnapshotDelegate> GetSnapshotDelegate() const;
 
+  //----------------------------------------------------------------------------
+  /// @brief      Add a view, implicit or not.
+  void AddView(int64_t view_id);
+
+  //----------------------------------------------------------------------------
+  /// @brief      Remove a view, implicit or not.
+  void RemoveSurface(int64_t view_id);
```

Comment:
Let's be consistent between the add/remove methods (`AddView`/`RemoveView` or `AddSurface`/`RemoveSurface`)

```suggestion
  void RemoveView(int64_t view_id);
```